carbon-design-system/carbon
# fix(react): uishell sidenavmenu active descendants

#### Changelog

**Changed**

- Updated the React SideNavMenu in UIShell so that it detects active descendant links when more than two levels of navigation are in the page heirarchy

#### Testing / Reviewing

- Create HeaderNavigation with three levels of nesting.
- Set a third-level link to 'isActive'
- Shrink the window so that the hamburger menu is displayed.
- Notice that the top level link in the side navigation is marked as active when the sub-link menu is not expanded.

Related to https://github.com/carbon-design-system/carbon/pull/14942
